NATIVE-221: Upgrade to RN 0.60.6

klinzo opened the pull request.

Schedulaar commented on .flowconfig:
```diff
@@ -11,18 +11,26 @@
 ; Ignore duplicate module providers
```
Can you add a link where you got the changes from or did you manually change it? :)

klinzo replied:
https://react-native-community.github.io/upgrade-helper/?from=0.59.9&to=0.60.6

klinzo commented:
Fixed warnings for `rn-fetch-blob` and `react-native-vector-icons`. Created an issue for `react-native-sentry` since an upgrade includes breaking changes. Fixed linking.

klinzo commented:
Regarding the warnings `componentWillReceiveProps` I'd just wait if these are resolved after upgrading all packages.

maxammann commented on android/app/src/main/java/tuerantuer/app/integreat/MainApplication.java:
```diff
@@ -1,28 +1,14 @@
-
-import com.RNFetchBlob.RNFetchBlobPackage;
-import com.dylanvann.fastimage.FastImageViewPackage;
+import cl.json.ShareApplication;
```
Unused import

klinzo replied:
Is used, MainApplication implements it

maxammann commented on ios/Podfile:
```diff
@@ -1,13 +1,39 @@
+  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
+  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
+
+  pod 'rn-fetch-blob', :path => '../node_modules/rn-fetch-blob'
```
Probably not needed
